.babelrc

```diff
@@ -1,3 +1,9 @@
 {
-    "presets": ["@babel/preset-env", "@babel/preset-react"]
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties"
```

Comment:
What does this babel plugin do?

Reply:
It allows the arrow functions, if you remove it, you may not be able to compile